lib/constants.js

```diff
@@ -46,6 +46,23 @@ module.exports.STEPPER = Object.freeze({
   }
 });
 
+module.exports.SERIAL_MODES = {
+  READ_CONTINUOUS: 0x00,
+  STOP_READING: 0x01
+};
+
+// ids for hardware and software serial ports on the board
+module.exports.SERIAL_PORT_IDs = {
```

Comment:
There are also the constants that map UART pins: https://github.com/jgautier/firmata/blob/master/lib/firmata.js#L482-L492